.github/PULL_REQUEST_TEMPLATE/05-fallback.md

```diff
@@ -0,0 +1,37 @@
+---
+
+name: 🤔 Unclear purpose or motivation
+about: >-
+  Submit a change to the ansible-core projects letting the maintainers judge
+  if it's desired. Only use if unsure and *only* if other types don't fit.
+title: Add a change for unspecified reason
```

Comment:
Can we omit title from the templates? It seems like people might not bother filling them out, leading to duplicates and very-similarly named PRs, making it more difficult to search for specific PRs. Generating one from the branch name or commit seems more helpful to me.

Reply:
Apparently, this metadata is not actually used by GH and the repo where I saw this dropped it in a follow-up: https://github.com/python/peps/pull/2998. So I'll delete these sections entirely too.